submissions/eoneoff/weather-api/manageCities.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+const fs = require('fs');
+
+function saveRecent (city) {

```

Comment:
You can remove nested if by using negative check and then return

Reply:
Ah, I can't see how. I need to remove the last record from recent if there are more then 9 records and save the changes anyway. If I return the saving will be skipped.
I have removed the nested if another way.

Reply:
I may be writing not-working code, but if it works, then its solution would be better
```suggestion
function saveRecent (city) {
 const recent = readRecent();
 if(!recent.includes(city)) return;
 recent.unshift(city);
 if (recent.lenght > 9) recent.pop();
 writeRecent(recent);
}
}
```

Reply:
So I have done. One thing: not
`if(!recent.includes(city)) return`
but
`if(recent.includes(city)) return`
because we add a new city to recent if it is not in the list

Actually I'm not sure, that it is correct to do so, because thus we make code somewhat shorter but, in my opinion less readable. This can make some sense in browser js where shorter code means less traffic, but in server node.js code it makes no sense. IMHO.

But there is another problem. Semistandard finds an error in my code, but the error message encoding is a mess and I can not get where the error is. The semistandard  extension for my VS Code shows no linter errors.